components/camel-file/src/main/java/org/apache/camel/component/file/FileOperations.java

```diff
@@ -157,23 +157,29 @@ public boolean buildDirectory(String directory, boolean absolute) throws Generic
         File endpointPath = endpoint.getFile();
         File target = new File(directory);
 
+        // check if directory is a path
+        boolean isPath = Math.max(directory.lastIndexOf("/"), directory.lastIndexOf("\\")) != -1;
```

Comment:
Why are you using lastIndexOf? Maybe you can just use contains to check if there is a slash?

Reply:
Sorry, makes sense. Had used the lastIndexOf position first before changing to boolean. Overlooked in refactoring. I have changed it now.